### DISCUSSION TITLE
[BUG]请问下  cf 内部 api 发送邮件 这个是在哪里

### DISCUSSION THREAD

**Discussion by oceanz764** [dreamhunter2333/cloudflare_temp_email#388] [General]
我添加了 发送邮件还是提示我 400 Failed to send mail Please enable resend or verified address list
![image](https://github.com/user-attachments/assets/f46e5279-f48a-441a-84c5-4fac9750f500)
![image](https://github.com/user-attachments/assets/65a2bd16-12e1-44a5-8c7a-0a3185c79c06)

**Comment by dreamhunter2333** [comment]
> 我添加了 发送邮件还是提示我 400 Failed to send mail Please enable resend or verified address list
> 
> ![image](https://github.com/user-attachments/assets/f46e5279-f48a-441a-84c5-4fac9750f500)
> 
> ![image](https://github.com/user-attachments/assets/65a2bd16-12e1-44a5-8c7a-0a3185c79c06)
> 
> 

只有在cf添加的验证地址才可以用 cf 发送邮件

**Comment by oceanz764** [comment]
是填这个嘛
<img width="989" alt="image" src="https://github.com/user-attachments/assets/ca26df6f-2480-4aea-95a9-f7ebc251c2d2">
 或者 怎么看在cf添加的验证地址

**Comment by dreamhunter2333** [comment]
> 是填这个嘛
> 
> <img width="989" alt="image" src="https://github.com/user-attachments/assets/ca26df6f-2480-4aea-95a9-f7ebc251c2d2">
> 
>  或者 怎么看在cf添加的验证地址

目标地址，我有空更新下文档， cf 的页面貌似改过

**Comment by oceanz764** [comment]
感恩 我的大佬

**Comment by oceanz764** [comment]
大佬 我发送的时候还是提示404
<img width="1060" alt="image" src="https://github.com/user-attachments/assets/b6e1f768-42ff-4b67-b4b9-8abf595bb326">
cf 变量 
<img width="663" alt="image" src="https://github.com/user-attachments/assets/ac2534df-1f44-4f8d-94eb-ec114709d604">
cf 邮箱验证
<img width="1010" alt="image" src="https://github.com/user-attachments/assets/890b26ec-8a1c-4011-a8ae-3728e5ad9511">
发件箱设置
<img width="1142" alt="image" src="https://github.com/user-attachments/assets/d7429cee-56cc-4bcb-8f90-c3bedc96fe49">

**Comment by dreamhunter2333** [comment]
SEND_MAIL 不是环境变量, 不在 var 中，需要绑定

**Comment by dreamhunter2333** [comment]
<img width="649" alt="image" src="https://github.com/user-attachments/assets/1ddb4306-122b-4b04-b5ca-4b69b60218ed">

**Comment by oceanz764** [comment]
在cf 上 怎么设置 大佬

**Comment by dreamhunter2333** [comment]
> 在cf 上 怎么设置 大佬

目前不清楚怎么通过用户界面设置，文档没有提到
